packages/shared/src/hooks/feed/useAutoRotatingAds.ts

```diff
@@ -75,7 +77,7 @@ export const useAutoRotatingAds = (
     return newAd;
   }, [feedIndex, index, logEventEnd, queryClient, queryKey]);
 
-  useQuery<Ad>({
+  const { refetch } = useQuery<Ad>({
```

Comment:
The refetch also resets the timer if applicable?

Reply:
Yeah